src/pages/DetailsPage.js

```diff
@@ -65,132 +66,142 @@ const getPhoneNumber = (details) => {
     return Str.removeSMSDomain(details.login);
 };
 
-const DetailsPage = (props) => {
-    const details = props.personalDetails[props.route.params.login];
-    if (!details) {
-        // Personal details have not loaded yet
-        return <FullscreenLoadingIndicator />;
+class DetailsPage extends React.PureComponent {
+    componentDidMount() {
+        if (lodashGet(this.props.route.params, 'login')) {
+            return;
+        }
+
+        // Leave the page when the login information is not available
+        Navigation.dismissModal();
     }
-    const isSMSLogin = Str.isSMSLogin(details.login);
 
-    // If we have a reportID param this means that we
-    // arrived here via the ParticipantsPage and should be allowed to navigate back to it
-    const shouldShowBackButton = Boolean(props.route.params.reportID);
-    const timezone = DateUtils.getLocalMomentFromTimestamp(props.preferredLocale, null, details.timezone.selected);
-    const GMTTime = `${timezone.toString().split(/[+-]/)[0].slice(-3)} ${timezone.zoneAbbr()}`;
-    const currentTime = Number.isNaN(Number(timezone.zoneAbbr())) ? timezone.zoneAbbr() : GMTTime;
-    const shouldShowLocalTime = !ReportUtils.hasExpensifyEmails([details.login]);
+    render() {
+        const details = this.props.personalDetails[lodashGet(this.props.route.params, 'login')];
+        if (!details) {
+            // Personal details have not loaded yet
+            return <FullscreenLoadingIndicator />;
+        }
+        const isSMSLogin = Str.isSMSLogin(details.login);
 
-    let pronouns = details.pronouns;
+        // If we have a reportID param this means that we
+        // arrived here via the ParticipantsPage and should be allowed to navigate back to it
+        const shouldShowBackButton = Boolean(this.props.route.params.reportID);
```

Comment:
Also this is quite interesting... In ROUTES.js we only have `getDetailsRoute` to construct this page's route, and we can only pass a Login here at the moment, no reportID - so how is `reportID` still available here??

At the moment I don't see the back button being shown on the details page anywhere... I thinkkkk the `reportID` you're looking for is in the `ReportParticipantsPage`, not `DetailsPage`